### ISSUE TITLE
Dialog A11y Pass

### ISSUE THREAD

**Issue by awalia13** [OfficeDev/office-ui-fabric-react#8061]
Tracking all of the issues brought up with the 2/20 accessibility pass on Dialog control
Details on individual bugs can be found on [Sharepoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7B8704792b-ff92-40ce-969f-93c02c8b96a9%7D&action=default&uid=%7B8704792B-FF92-40CE-969F-93C02C8B96A9%7D&ListItemId=1824&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble)
- [ ] Bug1: [Accessibility][Programmatic access]: ARIA attributes must conform to valid values
- [ ] Bug2: [Accessibility][Keyboard Navigation]: Focus indicator is not visible on the option B radio button in All emails together dialog for first time only.
- [ ] Bug3: [Accessibility][Usable]: All emails together dialog does not close with ESC in scan mode of narrator
- [ ] Bug4: [Accessibility][Programmatic access]:  Inappropriate name defined for All emails together dialog which appears on pressing enter on open dialog button under Default Dialog
- [ ] Bug5: [Accessibility][Usable]: Focus moves out of the dialog(to the Microsoft logo) upon pressing CTRL+Home instead of moving to the first element of the dialog
- [ ] Bug6: [Accessibility][Usable]: In Scan mode of narrator focus moves to the day/month/week image but it does not narrate anything.
- [ ] Bug7: [Accessibility][Usable]: Screen reader is not reading the entire content of dialog box  which appears on pressing enter on open dialog button under Blocking Dialog when the dialog opens.
- [ ] Bug8: [Accessibility][Usable]: Screen reader is not reading the entire content of dialog box  when the dialog opens.
- [ ] Bug9: [Accessibility][Usable]: Screen does not scroll down along with keyboard focus in Backward navigation.

**Comment by Kumar2608** [comment]
Bug6: [Accessibility][Usable]: In Scan mode of narrator focus moves to the day/month/week image but it does not narrate anything.
Now In scan mode narrator focus is not moving to day/month/week image.
Issue is fixed. Verified in the below build:
OS Build: version 1903(os build: 18956.1000)
URL: https://uifabric-prod.azurewebsites.net/?example=jhkjhkj#/components/dialog 

 
 Bug7: [Accessibility][Usable]: Screen reader is not reading the entire content of dialog box which appears on pressing enter on open dialog button under Blocking Dialog when the dialog opens.
 Here it is reading the content of the dialog box.
OS Build: version 1903(os build: 18956.1000)
URL: https://uifabric-prod.azurewebsites.net/?example=jhkjhkj#/components/dialog 

 Bug8: [Accessibility][Usable]: Screen reader is not reading the entire content of dialog box when the dialog opens.
 Here it is reading the content of the dialog box.
OS Build: version 1903(os build: 18956.1000)
URL: https://uifabric-prod.azurewebsites.net/?example=jhkjhkj#/components/dialog 
 
 Bug9: [Accessibility][Usable]: Screen does not scroll down along with keyboard focus in Backward navigation.
Here screen reader scroll down along with keyboard focus in backward navigation.
OS Build: version 1903(os build: 18956.1000)
URL: https://uifabric-prod.azurewebsites.net/?example=jhkjhkj#/components/dialog

**Comment by jdhuntington** [comment]
4 - Appears to have been fixed (demo is different, but label is correctly narrated now)

**Comment by jdhuntington** [comment]
1 - no repro

**Comment by jdhuntington** [comment]
2 - no repro (outline does appear if tab was used to open dialog, but does not appear otherwise)

**Comment by jdhuntington** [comment]
3 - Needs a comprehensive fix; will split into separate ticket.

**Comment by jdhuntington** [comment]
5 - no repro

**Comment by 999-Dev** [comment]
New issue found:

Bug10: [Accessibility][Tracking of focus]: Narrator focus moves to the background while navigating the dialogue box by using up and down arrow keys.

**Test Environment:** 
OS: Version: RS5_Prerelease(1903 OS Build 18334.1) 
URL: https://fabricweb.z5.web.core.windows.net/oufr/7.34.0/#/examples/dialog
Edge Browser Version: 
Microsoft Edge: 44.18983.1.0
Microsoft EdgeHTML: 18.18983
Chrome Browser Version:
Chrome Version: 76.0.3809.132

**Comment by jdhuntington** [comment]
Opened separate items instead of several issues being in 1 issue. Please refer to linked issues.
